docs/onboarding/09-exposing-twins-over-graphql.md

```diff
@@ -0,0 +1,46 @@
+# Chapter 9: Exposing Digital Twins over GraphQL
+
+- [Why GraphQL?](https://www.apollographql.com/docs/intro/benefits/) - Explains the value proposition of GraphQL
+- [GraphQL Tutorial](https://www.apollographql.com/docs/tutorial/introduction/) - Walks through coding a GraphQL service. **STOP after [Chapter 4 (Write Mutation Resolvers)](https://www.apollographql.com/docs/tutorial/mutation-resolvers/)**. Apollo Explorer is not necessary to complete the tutorial. Browsing to localhost address provides means to execute requests.
+
+## Key Points
+
+- GraphQL is a declarative model for exposing data from multiple sources over a common protocol.

```

Comment:
Could we break this down a bit?

Reply:
added a bit more explanation but would like to keep this succinct. Deeper explanation can be had during team discussions.